components/cam/chem_proc/src/Makefile

```diff
@@ -140,11 +140,15 @@ ifeq ($(COMPILER),xlf95)
   FFLAGS      := -g -c -qarch=auto -qnosave -qfree=f90 -qmoddir=$(OBJ_DIR) -I $(OBJ_DIR) -qstrict -O3
 endif
 ifeq ($(COMPILER),gfortran)
-  FFLAGS      := -g -c -ffree-form
+# FFLAGS      := -g -c -ffree-form
+  FFLAGS      := -g -c -ffree-form -J $(OBJ_DIR)
```

Comment:
I do not know about this `-J` flag. The code works without this new addition. Is there a reason this was added?

Reply:
The -J flag controls where the intermediate compiled files go.   Without the -J flag they clogged up the src directory.  The -J flag puts them in a separate directory $(OBJ_DIR), which I think has been the historical place for them.  Hence, it is not strictly necessary, but it is valuable and increases consistency across different compilers without any downside.  Hence, I recommend we keep it in the PR.

Reply:
Thanks! It works for gfortran but it doesn't do anything if we use Intel. Line 143 above (`# FFLAGS      := -g -c -ffree-form`) is not doing anything. Is it okay if I remove it?

Reply:
Yes, you can remove the commented line.  I was keeping it while I was testing new options.  Since I ended up just adding an extra option (-J) there seems little value in keeping the original line around.  Can you remove it?  Or do you need me to do it?

Reply:
I just removed the commented out line, for clarity.

Reply:
I just pushed a commit to remove the commented-out line.